### PR TITLE
Samar e commerce website

### PR DESCRIPTION
**Pull request by samar4tech**

### CODE REVIEW

File: tasks-by-users/samar4tech/01-e-commerce-website/style.css [hunk 1]
```diff
@@ -0,0 +1,508 @@
...
+  margin: 25px 0;
+}
+.btn {
+  display: inline-block;
+  background-color: #ff523b;
+  color: #fff;
+  padding: 8px 30px;
+  margin: 30px 0;
+  border-radius: 30px;
+  transition: background 0.5s;
```
Comment (Rehmatulla-Khan) [inline tasks-by-users/samar4tech/01-e-commerce-website/style.css:77]: grp these properties 
CC: @mmusk @Abbas4tech @Naufil-Shemle

File: tasks-by-users/samar4tech/01-e-commerce-website/style.css [hunk 2]
```diff
@@ -0,0 +1,508 @@
...
+}
+.header {
+  background: radial-gradient(#fff, #ffd6d6);
+}
+.header .row {
+  margin-top: 70px;
+}
+.btn:hover {
+  background-color: #563434;
```
Comment (Rehmatulla-Khan) [inline tasks-by-users/samar4tech/01-e-commerce-website/style.css:86]:
```suggestion
  background: #563434;
```

File: tasks-by-users/samar4tech/01-e-commerce-website/style.css [hunk 3]
```diff
@@ -0,0 +1,508 @@
...
+  color: #8a8a8a;
+}
+.footer h3 {
+  color: #fff;
+  margin-bottom: 20px;
+}
+.footer-col-1,
+.footer-col-2,
+.footer-col-3,
+.footer-col-4 {
```
Comment (Rehmatulla-Khan) [inline tasks-by-users/samar4tech/01-e-commerce-website/style.css:223]: can you use attribute selector here ? @samar4tech

File: tasks-by-users/samar4tech/01-e-commerce-website/style.css [hunk 4]
```diff
@@ -0,0 +1,508 @@
...
+    left: 0;
+    width: 100%;
+
+    overflow: hidden;
+    transition: max-height 0.5s;
+  }
+  nav ul li {
+    margin-right: 50px;
+    margin-top: 10px;
+    margin-bottom: 10px;
```
Comment (Rehmatulla-Khan) [inline tasks-by-users/samar4tech/01-e-commerce-website/style.css:284]:
```suggestion
    margin: 10px 50px 10px 0;
```

File: tasks-by-users/samar4tech/01-e-commerce-website/account.html [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+  </head>
+  <body>
+    <div class="container">
+      <div class="navbar">
+        <div class="logo">
+          <a href="index.html"><img src="images/logo.png" /></a>
```
Comment (Naufil-Shemle) [inline tasks-by-users/samar4tech/01-e-commerce-website/account.html:24]: insert `alt`

File: tasks-by-users/samar4tech/01-e-commerce-website/style.css [hunk 6]
```diff
@@ -0,0 +1,508 @@
...
+}
+body {
+  font-family: "Poppins", sans-serif;
+}
+
+.container {
+  max-width: 1300px;
+  margin: auto;
+  padding-left: 25px;
+  padding-right: 25px;
```
Comment (Naufil-Shemle) [inline tasks-by-users/samar4tech/01-e-commerce-website/style.css:14]: use short-hands

File: tasks-by-users/samar4tech/01-e-commerce-website/productdetails.html [hunk 7]
```diff
@@ -0,0 +1,247 @@
...
+            <i class="fa fa-star"></i>
+            <i class="fa fa-star-o"></i>
+          </div>
+          <p>$50.00</p>
+        </div>
+      </div>
+    </div>
+
+    <!-------------footer----------------->
+    <div class="footer">
```
Comment (Abbas4tech) [inline tasks-by-users/samar4tech/01-e-commerce-website/productdetails.html:176]: Use `footer` tag @samar4tech

File: tasks-by-users/samar4tech/01-e-commerce-website/style.css [hunk 8]
```diff
@@ -0,0 +1,508 @@
...
+  margin-bottom: 30px;
+}
+.col-3 img {
+  width: 100%;
+}
+.small-container {
+  max-width: 1080px;
+  margin: auto;
+  padding-left: 25px;
+  padding-right: 25px;
```
Comment (Naufil-Shemle) [inline tasks-by-users/samar4tech/01-e-commerce-website/style.css:103]: here also use short-hands

File: tasks-by-users/samar4tech/01-e-commerce-website/product.html [hunk 9]
```diff
@@ -0,0 +1,271 @@
...
+        <span>1</span>
+        <span>2</span>
+        <span>3</span>
+        <span>4</span>
+        <span>&#8594;</span>
+      </div>
+    </div>
+
+    <!-------------footer----------------->
+    <div class="footer">
```
Comment (Abbas4tech) [inline tasks-by-users/samar4tech/01-e-commerce-website/product.html:217]: Use `footer` tag @samar4tech

File: tasks-by-users/samar4tech/01-e-commerce-website/style.css [hunk 10]
```diff
@@ -0,0 +1,508 @@
...
+  color: #555;
+}
+/* <---------testimonial-------------- > */
+.testimonial {
+  padding-top: 100px;
+}
+.testimonial .col-3 {
+  text-align: center;
+  padding: 40px 20px;
+  box-shadow: 0 0 20px 0px rgba(0, 0, 0, 0.1);
```
Comment (Abbas4tech) [inline tasks-by-users/samar4tech/01-e-commerce-website/style.css:165]: What is this `20px` stands for ???
How it will effect the shadow ???
@samar4tech

**Review by Abbas4tech — CHANGES_REQUESTED** [review]

File: tasks-by-users/samar4tech/01-e-commerce-website/index.html [hunk 11]
```diff
@@ -0,0 +1,375 @@
...
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+  </head>
+  <body>
+    <div class="header">
```
Comment (Abbas4tech) [inline tasks-by-users/samar4tech/01-e-commerce-website/index.html:21]: Use `header` tag @samar4tech

**Review by Abbas4tech — CHANGES_REQUESTED** [review]

File: tasks-by-users/samar4tech/01-e-commerce-website/style.css (hunk 2 above)
Comment (samar4tech) [reply tasks-by-users/samar4tech/01-e-commerce-website/style.css:103]: specifically used only color property in the bg so opted background-color.Will take care from the next time.

File: tasks-by-users/samar4tech/01-e-commerce-website/account.html [hunk 12]
```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
```
Comment (faran4engg) [inline tasks-by-users/samar4tech/01-e-commerce-website/account.html:1]: can you explain why we use `DOCTYPE` ?

**Review by faran4engg — APPROVED** [review]

File: tasks-by-users/samar4tech/01-e-commerce-website/style.css (hunk 10 above)
Comment (samar4tech) [reply tasks-by-users/samar4tech/01-e-commerce-website/style.css:211]: 20px stands for blur radius here.
it affects the shadow as "the higher the no,the more blurred the shadow will be".

File: tasks-by-users/samar4tech/01-e-commerce-website/account.html (hunk 12 above)
Comment (samar4tech) [reply tasks-by-users/samar4tech/01-e-commerce-website/account.html:1]: It's an instruction to the browser about what version of HTML, basically the page is written in.

File: tasks-by-users/samar4tech/01-e-commerce-website/index.html (hunk 11 above)
Comment (samar4tech) [reply tasks-by-users/samar4tech/01-e-commerce-website/index.html]: done

File: tasks-by-users/samar4tech/01-e-commerce-website/product.html (hunk 9 above)
Comment (samar4tech) [reply tasks-by-users/samar4tech/01-e-commerce-website/product.html]: done

File: tasks-by-users/samar4tech/01-e-commerce-website/account.html (hunk 5 above)
Comment (samar4tech) [reply tasks-by-users/samar4tech/01-e-commerce-website/account.html]: done

File: tasks-by-users/samar4tech/01-e-commerce-website/style.css (hunk 4 above)
Comment (samar4tech) [reply tasks-by-users/samar4tech/01-e-commerce-website/style.css]: done

**Review by shaikh-7abish — APPROVED** [review]

**Review by mujeeb100 — APPROVED** [review]

**Review by Abbas4tech — CHANGES_REQUESTED** [review]
why card hover project files are added here ? @samar4tech

**Review by Abbas4tech — APPROVED** [review]
👍

File: tasks-by-users/samar4tech/01-e-commerce-website/style.css (hunk 3 above)
Comment (samar4tech) [reply tasks-by-users/samar4tech/01-e-commerce-website/style.css]: done

**Pull request merged**